packages/platform-sdk-ada/src/services/client.ts

```diff
@@ -32,11 +32,16 @@ export class ClientService implements Contracts.ClientService {
 	}
 
 	public async transaction(id: string): Promise<Contracts.TransactionDataType> {
-		throw new Exceptions.NotImplemented(this.constructor.name, "transaction");
+		const walletId = `98c83431e94407bc0889e09953461fe5cecfdf18`; // FIXME
```

Comment:
@faustbrian Where could I get this wallet id from? This is what we were discussing yesterday. I think it should be stored locally in the wallet metadata (profile?) once we create the Cardano wallet. For that, we send the account public key and we get back that id. Any chance we have any place for storing it and using it here?

Reply:
That ID would be handled outside of this package so I'll make a separate task for this. Is it impossible to get the transaction without also knowing the wallet ID?

Reply:
The endpoint requires both, transaction is a 'subresource' of wallet.
`https://localhost:8090/v2/wallets/{walletId}/transactions/{transactionId}`

Reply:
I'll take a look at this tomorrow to see if there is a way to get the transaction without knowing the wallet ID because this method should work without any input except the transaction ID.

Reply:
Doesn't look like there is a way so I'll leave this as a draft until I figure out how to restructure the contracts to allow for optional parameters like the wallet ID because other coins probably will also have some weird things like that.